src/main/java/org/cloudfoundry/promregator/scanner/ReactiveAppInstanceScanner.java

```diff
@@ -182,9 +183,11 @@ public void setNumberOfInstances(int numberOfInstances) {
 				// NB: This drops the current target!
 				return Mono.empty();
 			}
-			
-			v.setApplicationId(sas.getId());
-			
+
```

Comment:
Since I remove the id assignment. I wasn't certain where to place this warning. I placed it a few lines down where we fetch the urls from the SAS. I'm fine removing it if you like.